deployments/scripts/protobuf-codegen

```diff
@@ -16,7 +16,7 @@ done
 echo "Pulling proto dependencies from BSR..."
 pushd "${repo_root}/proto/"
 # Make sure the lockfile is up to date.
-buf dep update penumbra
+buf mod update penumbra
```

Comment:
it'd be nice to pull this into a separate commit, so that this commit only does what it says on the tin: add a field to `TransactionPlannerRequest`

Reply:
> why did this change? i was a little surprised to see a change to the `protobuf-codegen` logic in this commit

since `buf dep update` was [depreciated](https://github.com/bufbuild/buf/releases)

Reply:
But your link says the opposite of that.

https://github.com/bufbuild/buf/releases/tag/v1.32.0

> - Move `buf mod update` to `buf dep update`. `buf mod update` is now deprecated.

Reply:
must have misread that! will fix, thanks

Reply:
> But your link says the opposite of that.
> 
> https://github.com/bufbuild/buf/releases/tag/v1.32.0
> 
> > * Move `buf mod update` to `buf dep update`. `buf mod update` is now deprecated.

https://github.com/penumbra-zone/penumbra/pull/4626

Reply:
i'm disappointed that this review feedback wasn't addressed before merging. in the future, we should resolve our peers' comments before merging code into `main`.

Reply:
> i'm disappointed that this review feedback wasn't addressed before merging. in the future, we should resolve our peers' comments before merging code into `main`.

yes, my mistake! I accidentally misread the tagged released, but I opened another PR to fix this. I will be extra cautious going forward